tests/models/multiple_choice_qa/multiple_choice_tuple_inference_model_test.py

```diff
@@ -0,0 +1,29 @@
+# pylint: disable=no-self-use,invalid-name
+from unittest import TestCase
+import os
+import shutil
+
+from deep_qa.models.multiple_choice_qa.multiple_choice_tuple_inference import \
+    MultipleChoiceTupleInferenceModel
+from ...common.constants import TEST_DIR
+from ...common.models import get_model, write_multiple_choice_tuple_inference_files
+
+class TestMultipleChoiceTupleInferenceModel(TestCase):
+    def setUp(self):
+        os.makedirs(TEST_DIR, exist_ok=True)
+        write_multiple_choice_tuple_inference_files()
+
+    def tearDown(self):
+        shutil.rmtree(TEST_DIR)
+
+    def test_train_does_not_crash(self):
```

Comment:
Can you test loading, too?  The test ends up looking like this: https://github.com/allenai/deep_qa/blob/942d66980b589d924e07bc127f0c38e4a7c89352/tests/models/multiple_choice_qa/multiple_true_false_memory_network_test.py#L26-L35.

Making sure this works is really important for actually running this in the evaluation framework, which you should be able to do very soon.